src/openforms/appointments/contrib/jcc/client.py

```diff
@@ -1,7 +1,9 @@
+from zeep.client import Client
+
 from .models import JccConfig
 
 
-def get_client():
+def get_client() -> Client:
     service = JccConfig.get_solo().service
     assert service is not None
     return service.build_client()
```

Comment:
I remember this comment from Chris about the asserts: https://github.com/open-formulieren/open-forms/pull/3024#discussion_r1177632873

Maybe the client should be checked and in case of `None` raise an informative error like 'service not configured' or something?

Reply:
in principle we have the config check page for this to guard against it. Whether this crashes in production with an `AssertionError` or an `AttributeError` (NoneType has no attribute `build_client`) has the same result in the end - it crashes.